util/Radiance_Monitor/build_RadMon_cmake.sh

```diff
@@ -3,139 +3,75 @@
 #------------------------------------------------------------------
 #  build_RadMon_cmake.sh
 #
-#  This script builds all of the executables in the 
-#  nwprod/radmon_shared/exec, data_extract/exec, and image_gen/exec
-#  subdirectories.  
-#
-#  The operational RadMon executables (in nwprod/radmon_shared/exec) 
-#  may also be built as part of the whole GSI package.  To do this 
-#  ensure BUILD_UTIL=ON when running cmake or use the 
-#  ProdGSI/ush/build_all_cmake.sh script.
+#  This script builds all of the RadMon executables and moves them
+#  to nwprod/radmon_shared/exec, data_extract/exec, and 
+#  image_gen/exec locations. 
 #------------------------------------------------------------------
-set -ax
-
-mode=${1:-}
-MY_RADMON=${2:-}
-
-top_level=${PWD}
-echo "top_level = ${top_level}"
-
-export MY_RADMON=${MY_RADMON:-$top_level}
-echo "MY_RADMON = ${MY_RADMON}"
-
-target=`./get_machine.sh`
-echo "target = $target"
-
-if [[ $target = "wcoss_c" || $target = "wcoss_d" ||
-      $target = "orion"   || $target = "wcoss2"  || 
-      $target = "s4" ]] ; then
-    . $MODULESHOME/init/sh
-elif [[ $target = "hera" ]] ; then
-    . /apps/lmod/lmod/init/sh
-elif [[ $target = "jet" ]] ; then
-    . /apps/lmod/lmod/init/sh
-elif [[ -d /lfs && -d /dfs ]]; then
-    . $MODULESHOME/init/bash
-    target=wcoss2
-else
-    echo "unknown target = $target"
-    exit 9
-fi
+RAD_ROOT=${PWD}
+echo "RAD_ROOT = ${RAD_ROOT}"
+
+# Get the root of the cloned GSI directory
+readonly GSI_ROOT=$(cd "$(dirname "$(readlink -f -n "${BASH_SOURCE[0]}" )" )/../.." && pwd -P)
+
+echo "GSI_ROOT = ${GSI_ROOT}"
+
+#-------------------------------------------------------
+# Modify CoMakeLists.txt to turn off GSI and ENKF builds
+#-------------------------------------------------------
+cml=${GSI_ROOT}/CMakeLists.txt
+cp ${cml} ${cml}.orig
```

Comment:
Do not modify `CMakeLists.txt` or move them. Please use `cmake` options to turn `ON|OFF`